src/dlgProfilePreferences.cpp

```diff
@@ -1779,3 +1804,37 @@ void dlgProfilePreferences::slot_script_selected(int index)
         preview->setText(mpHost->getActionUnit()->getAction(itemId)->getScript());
     }
 }
+
+/*!
+ * \brief dlgProfilePreferences::slot_changeShowSpacesAndTabs
```

Comment:
I don't know what is this and neither Qt Creator nor CLion render this syntax... what is this trying to do?